k8s/prod/common/ia/identity.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+ name: ia
+ namespace: ia
+spec:
+ type: 0
+ ResourceID: /subscriptions/8cbc6f36-7c56-4963-9d36-739db5d00b27/resourceGroups/managed-identities-prod-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/ia-prod-mi
```

Comment:
@timja can you help with subscriptions id? is it `ia-identity-object-id` which we have got on AKS workshops or it should stay as it is above?